app-load-params-db.json

```diff
@@ -397,6 +397,18 @@
     "curve": ["ed25519"],
     "path": ["44'/1955'"]
   },
+  "ZTG": {
+    "appFlags": {"nanos": "0x000", "nanos2": "0x000", "nanox": "0x200"},
+    "appName": "Zeitgeist",
+    "curve": ["ed25519"],
+    "path": ["44'/354'"]
+  },
+  "ZTG_XL": {
```

Comment:
just a question : are you sure that XL version targets the Nano **S** device ?

Reply:
XL versions of substrate apps only target LNS